src/components/layout/Navigation.js

```diff
@@ -82,7 +82,7 @@ class Navigation extends React.Component {
   getLoginOrProfileNav() {
     if (this.props.user.authenticated) {
       return (
-        <div>
+        <>
```

Comment:
if not necessary, `<NavLink>` can be the root element

Reply:
it's necessary because it returns multiple navlink elements